packages/lmi/tests/test_llms.py

```diff
@@ -58,6 +58,18 @@ def test_instantiation_methods(self) -> None:
         assert model4.config["model_list"][0]["litellm_params"]["temperature"] == 0.5
         assert model4.config["model_list"][0]["litellm_params"]["max_tokens"] == 100
 
+        # Test logprobs and top_logprobs configuration passing through (OpenAI-specific)
+        name = CommonLLMNames.OPENAI_TEST.value
+        config_with_logprobs = {
+            "logprobs": True, 
+            "top_logprobs": 20, 
```

Comment:
There's trailing whitespace here, why is that?

Reply:
where?